biovectors_modules/word2vec_analysis_helper.py

```diff
@@ -34,33 +34,70 @@ def generate_timeline(
     return timeline_df
 
 
-def get_global_distance(
-    year_one_model: str, year_two_model: str, shared_tokens: Sequence[str]
-) -> pd.DataFrame:
-    """
-    This function is designed to get the global distance of ever token between two years.
-    Global distance is basically the cosine similarity (token_year_one, token_year_two)
+def _calculate_distances(
+    year_one_model: pd.DataFrame,
+    year_two_model: pd.DataFrame,
+    neighbors: int,
+    input_queue: JoinableQueue,
+    output_queue: JoinableQueue,
+):
```

Comment:
Is this function replacing both the previous global and local functions? Maybe adding docstring here might help.

Reply:
Yeah forgot to add that. Will do in next commit